pages/home.py

```diff
@@ -29,6 +31,14 @@ def is_task_in_progress(self):
     def task_in_progress(self):
         return self.selenium.find_element(*self._task_in_progress_locator).text
 
+    @property
+    def displayed_suggested_first_tasks_list(self):
```

Comment:
Perhaps a better name for this would be something like `is_suggested_first_tasks_heading_visible`. This is going to return a `boolean` (i.e., either `True` or `False`) so using `is_` as a prefix on the property name gives a clue that a boolean is expected.